man/git-feature.md

```diff
@@ -40,7 +44,7 @@ git-feature(1) -- Create/Merge feature branch
     $ git features dependencies
     $ (features/dependencies) ...
     $ (features/dependencies) git checkout master
-    $ git features finish dependencies
+    $ git features finish --squash dependencies
```

Comment:
Maybe we can show both  commands with/without `squash` in the example?
Like `git features finish dependencies # Or 'git features finish --squash dependencies'`

And it would be better if you can compare the default merge mode with the squash one in the  documentation.